doc/arbtt.xml

```diff
@@ -318,6 +318,20 @@ $ runhaskell Setup.hs install</screen>
       parentheses.
     </para>
 
+    <para>
+      You can assign conditions to variables using
```

Comment:
Better say

> You can define short-hand names for conditions using

to not needlessly confuse people who think of variables as mutable boxes.

Reply:
done